### ISSUE TITLE
Special characters (æøå) in file and folder names

### ISSUE THREAD

**Issue by esbennn** [owncloud/core#26375]
Hello!
I recently moved my office from Dropbox to ownCloud. I have set up a Raspberry Pi (raspbian jessie) with an external hdd, which is accessed via our dns-server. It works and performs quite well!
However, once in a while we noticed that specific files or entire folders would not sync from desktop to server, causing the client to halt the sync, thereby not getting other new files. 
It turned out that it happens when a file or folder containing æ, ø or å (special danish letters) is created. The sync client gets the red cross, and it says "server replied: Forbidden".
Of course, we could just get used to not using these native characters, but it just easily happens by mistake. Besides, there are already an unknown amount of files and folders containing these letters in our Dropbox, which we wish to move.
I wondered if it had to do with the locale settings on the Pi , but those are exactly the same as on my desktop (Fedora 24), and it accepts folder/filenames with æøå perfectly fine.
I cant seem to find any documentation anywhere, that states æøå aren't allowed.

### Steps to reproduce
1. Create file or folder with a name containing æ, ø or å.
2. Wait for client to sync
3.

### Expected behaviour
Client should sync without errors

### Actual behaviour
Client halts syncing, stating "server replied: Forbidden"

### Server configuration
**Operating system**:  Linux raspberrypi 4.4.13-v7+ #894 SMP Mon Jun 13 13:13:27 BST 2016 armv7l GNU/Linux

**Web server:** Apache/2.4.10 (Raspbian)

**Database:** Ver 14.14 Distrib 5.5.50

**PHP version:** PHP 5.6.24-0+deb8u1 (cli)

**ownCloud version:** 7.0.4+dfsg-4~deb8u4 (Debian) (stable) 

**Updated from an older ownCloud or fresh install:** Fresh install 

**Where did you install ownCloud from:** SUSE build service

**Signing status (ownCloud 9.0 and above):**

```
Login as admin user into your ownCloud and access 
http://example.com/index.php/settings/integrity/failed 
paste the results here.
```

**List of activated apps:**

```
If you have access to your command line run e.g.:
sudo -u www-data php occ app:list
from within your ownCloud installation folder
```

**The content of config/config.php:**

```
If you have access to your command line run e.g.:
sudo -u www-data php occ config:list system
from within your ownCloud installation folder

or 

Insert your config.php content here
(Without the database password, passwordsalt and secret)
```

**Are you using external storage, if yes which one:** local/smb/sftp/...

**Are you using encryption:** yes/no

**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...

#### LDAP configuration (delete this part if not used)

```
With access to your command line run e.g.:
sudo -u www-data php occ ldap:show-config
from within your ownCloud installation folder

Without access to your command line download the data/owncloud.db to your local
computer or access your SQL server remotely and run the select query:
SELECT * FROM `oc_appconfig` WHERE `appid` = 'user_ldap';


Eventually replace sensitive data as the name/IP-address of your LDAP server or groups.
```

### Client configuration
**Browser:**

**Operating system:**

#### Web server error log
```
Insert your webserver log here
```

#### ownCloud log (data/owncloud.log)
```
Insert your ownCloud log here
```

#### Browser log
```
Insert your browser log here, this could for example include:

a) The javascript console log
b) The network log 
c) ...
```

**Comment by RealRancor** [comment]
Hi,

> ownCloud version: 7.0.4+dfsg-4~deb8u4 (Debian) (stable) 

from the issue template you have filled out:

> Before reporting any issues please make sure that you're using the latest available version for your major branch (e.g. 9.0.x), see https://owncloud.org/changelog/

7.0.4 is outdated, end of life since longer time. Personally i think you can instantly close this issue here and install a recent 9.1.1 installation.

**Comment by PVince81** [comment]
OC does work with special characters, so likely a setup issue

**Comment by esbennn** [comment]
Thank you guys. I was not aware there was a newer version - i guess i just assumed raspbian had the newest version in it's repos. Shame on me ;)
I have now updated to 9.1.1.

Turns out that the issue was actually related to the external harddrive i use as ownclouds data folder.
It's formatted as vfat in order to make the disk compatible with windows pcs (just in case). Adding the option "utf8" to the mount in fstab solved that  particular issue.
